style.css

```diff
@@ -331,3 +345,96 @@
   height: 80px;
   border-radius: 50px;
 }
+
+.contact-form {
+  box-sizing: border-box;
+  display: flex;
+  flex-wrap: wrap;
+  margin: 10px 0;
+}
```

Comment:
- [ ] Kindly add background image as shown in the figma template to enhance your website look: 
![image](https://user-images.githubusercontent.com/91486791/186272051-d88fa6f7-2847-4db6-be6a-d4abb331ada1.png)